src/settings/toml/target.rs

```diff
@@ -13,7 +14,7 @@ use serde::{Deserialize, Serialize};
 pub struct Target {
     pub account_id: String,
     #[serde(rename = "kv-namespaces")]
-    pub kv_namespaces: Option<Vec<KvNamespace>>,
+    pub kv_namespaces: Option<Vec<ConfigKvNamespace>>,
```

Comment:
i think i would flip these, actually, and name the thing you attach to `Manifest` `ConfigKvNamespace`  and use just `KvNamespace`s for `Target`